%-------------------------------------------------------------------------------
%	SECTION TITLE
%-------------------------------------------------------------------------------
\cvsection{Projects}


%-------------------------------------------------------------------------------
%	CONTENT
%-------------------------------------------------------------------------------
\begin{cventries}

  \cventry
    {Weissman Lab} % Role
    {dynast} % Event
    {\faGithub\phantom{a}\href{https://github.com/aristoteleo/dynast-release}{aristoteleo/dynast-release}} % Location
    {Oct 2020 - Present} % Date(s)
    {
      \begin{cvitems} % Description(s)
        \item {Pipeline for complete splicing and labeling quantification from metabolic labeling scRNA-seq experiments.}
        \item {Ability to perform estimation of labeled/unlabeled RNA using a Bayesian inference model.}
      \end{cvitems}
    }


\cventry
  {Ginkgo Bioworks} % Role
  {Mass Spectrometry Pipeline} % Event
  {} % Location
  {Jun 2020 - Aug 2020} % Date(s)
  {
    \begin{cvitems} % Description(s)
      \item {Migrated mass spectrometry pipeline, which depended on an on-prem Windows machine, to Linux on AWS and Airflow.}
      \item {Implemented the ability to call Windows DLLs within Linux with Wine and IronPython.}
      \item {Retired an internal fork of ProteoWizard by contributing back to the project.}
    \end{cvitems}
  }

\cventry
  {Pachter Lab} % Role
  {Colosseum} % Event
  {\faGithub\phantom{a}\href{https://github.com/pachterlab/colosseum}{pachterlab/colosseum}} % Location
  {Sept 2019 - Jun 2020} % Date(s)
  {
    \begin{cvitems} % Description(s)
      \item {A low-cost, modular, open-source automated fluid sampling device for scalable microfluidic applications.}
      \item {Can be built for less than \$100 using off-the-shelf components in less than an hour.}
      \item {Developed its user interface using PyQt5 and installable from Pypi; later migrated to a fully-featured web application written with React.js.}
    \end{cvitems}
  }

\cventry
  {Pachter Lab} % Role
  {kb-python} % Event
  {\faGithub\phantom{a}\href{https://github.com/pachterlab/kb_python}{pachterlab/kb\_python}} % Location
  {Oct 2019 - Jun 2020} % Date(s)
  {
    \begin{cvitems} % Description(s)
      \item {A Python wrapper around the kallisto | bustools pipeline for scRNA-seq pre-processing.}
      \item {Pre-processes scRNA-seq data into gene count matrices and RNA velocity matrices.}
      \item {Provides pre-built reference indices for pseudoalignment.}
    \end{cvitems}
  }

\cventry
    {Ginkgo Bioworks}
    {NGS Pipeline}
    {}
    {Jun 2019 - Sept 2019}
    {
     \begin{cvitems} % Description(s)
       \item {Migrated in-house NGS pipeline to the cloud (AWS S3, Batch, DynamoDB).}
       \item {Implemented pipeline metrics collection with Elasticsearch and Grafana.}
       \item {Implemented contamination screening via taxonomy analysis with Centrifuge.}
      \end{cvitems}
    }

\cventry
    {Sternberg Lab}
    {\href{https://github.com/Lioscro/alaska-parse}{Alaska}}
    {\faGithub\phantom{a}\href{https://github.com/Lioscro/alaska-parse}{lioscro/alaska-parse}}
    {Jan 2017 - Aug 2019}
    {
     \begin{cvitems} % Description(s)
       \item Developed a framework for automated, complete RNA-seq analysis.
       \item Parse Server backend. Frontend portal in Javascript, HTML/CSS. Containerization with Docker.
       \item Presented a poster at the 21st International \textit{C. elegans} Conference.
      \end{cvitems}
    }

\end{cventries}
