%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Deedy - One Page Two Column Resume
% LaTeX Template
% Version 1.2 (16/9/2014)
%
% Original author:
% Debarghya Das (http://debarghyadas.com)
%
% Original repository:
% https://github.com/deedydas/Deedy-Resume
%
% IMPORTANT: THIS TEMPLATE NEEDS TO BE COMPILED WITH XeLaTeX
%
% This template uses several fonts not included with Windows/Linux by
% default. If you get compilation errors saying a font is missing, find the line
% on which the font is used and either change it to a font included with your
% operating system or comment the line out to use the default font.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% TODO:
% 1. Integrate biber/bibtex for article citation under publications.
% 2. Figure out a smoother way for the document to flow onto the next page.
% 3. Add styling information for a "Projects/Hacks" section.
% 4. Add location/address information
% 5. Merge OpenFont and MacFonts as a single sty with options.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% CHANGELOG:
% v1.1:
% 1. Fixed several compilation bugs with \renewcommand
% 2. Got Open-source fonts (Windows/Linux support)
% 3. Added Last Updated
% 4. Move Title styling into .sty
% 5. Commented .sty file.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% Known Issues:
% 1. Overflows onto second page if any column's contents are more than the
% vertical limit
% 2. Hacky space on the first bullet point on the second column.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\documentclass[]{resume}
\usepackage{fontawesome}
\usepackage{hyperref}
\usepackage{fancyhdr}

\pagestyle{fancy}
\fancyhf{}

\begin{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     LAST UPDATED DATE
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\lastupdated
\vspace*{0.5em}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     TITLE NAME
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\namesection{Kyung Hoi (Joseph)}{Min}{
Pasadena, CA | \faEnvelope\phantom{ } \href{mailto:phoenixter96@gmail.com}{phoenixter96@gmail.com} | \faGithub  \href{https://github.com/lioscro}{\phantom{ }lioscro} | \faLinkedin  \href{https://www.linkedin.com/in/lioscro/}{\phantom{ }lioscro}

}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     COLUMN ONE
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{minipage}[t]{0.28\textwidth}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     EDUCATION
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Education}
\subsection{MIT}
\descript{MS in EECS}
\location{Jun 2022 | Cambridge, MA}
\sectionsep

\subsection{Caltech}
\descript{BS in Biology and \\ Computer Science}
\location{Jun 2020 | Pasadena, CA}
% \location{Cum. GPA: 4.0 / 4.3}
%Major GPA: 3.9 / 4.0}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     LINKS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Links}
% Facebook:// \href{https://facebook/dd}{\bf dd} \\
% Github:// \href{https://github.com/deedydas}{\bf deedydas} \\
% LinkedIn://  \href{https://www.linkedin.com/in/debarghyadas}{\bf debarghyadas} \\
% YouTube://  \href{https://www.youtube.com/user/DeedyDash007}{\bf DeedyDash007} \\
% Twitter://  \href{https://twitter.com/debarghya_das}{\bf @debarghya\_das} \\
% Quora://  \href{https://www.quora.com/Debarghya-Das}{\bf Debarghya-Das}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     COURSEWORK
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Coursework}
% \subsection{Undergraduate}
% \location{Biology}
% Regulation of Gene Expression \\
% Cell Biology \\
% Genetics \\
% Bioinformatics {\footnotesize \textit{(Planned)}}\\
% \sectionsep
% \location{Computer Science}
% Programming Methods \\
% Decidability and Tractability \\
% Communication Networks \\
% Algorithms \\
% Learning systems {\footnotesize \textit{(machine learning)}}\\
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     SKILLS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Skills}
\subsection{Programming}
\textbullet{\textbf{Python}}\\
\textbullet{\textbf{Javascript}, Typescript}\\
\textbullet{Ruby on Rails}\\
\textbullet{SQL}\\
\textbullet{C\# (.NET Framework)}\\
\textbullet{C, C++}
\sectionsep

\subsection{Software}
\textbullet{\textbf{Git, CI/CD}}\\
\textbullet{\textbf{Docker}}\\
\textbullet{\textbf{Django}}\\
\textbullet{\textbf{Apache Airflow}}\\
\textbullet{\textbf{AWS (S3, Batch, RDS, Lambda)}}\\
\textbullet{\textbf{Nextflow}}\\
\textbullet{\textbf{React.js}}\\
\textbullet{Snowflake}\\
\textbullet{Kubernetes}\\
\textbullet{Kafka}\\
% \textbullet{Elasticsearch, Grafana}\\
% \textbullet{CI/CD}\\
% \textbullet{Celery, RabbitMQ}\\
% \textbullet{Numpy, Pandas}
\sectionsep

\subsection{Other}
\textbullet{Software design and architecture}\\
\textbullet{Technical communication}\\
\textbullet{Computational biology (NGS)}
\sectionsep

% \subsection{Molecular Biology}
% \textbullet{PCR}\\
% \textbullet{Cloning}\\
% \textbullet{Purification}\\
% \sectionsep



% \location{Over 5000 lines:}
% Java \textbullet{}   Shell \textbullet{} Python \textbullet{} Javascript \\
% OCaml \textbullet{} Matlab \textbullet{} Rails \textbullet{} \LaTeX\ \\
% \location{Over 1000 lines:}
% C \textbullet{} C++ \textbullet{} CSS \textbullet{} PHP \textbullet{} Assembly \\
% \location{Familiar:}
% AS3 \textbullet{} iOS \textbullet{} Android \textbullet{} MySQL
% \sectionsep

\section{Publications}
\renewcommand\refname{\vskip -1.5em} % Couldn't get this working from the .cls file
\begin{thebibliography}{9}
\bibitem{spateo}
Spateo: multidimensional spatiotemporal modeling of single-cell spatial transcriptomics. in press, 2024
\bibitem{dynast}
Dynast: Inclusive and efficient quantification of metabolically labeled transcripts in single cells. MIT, 2022
% \bibitem{kp}
% Lineage tracing reveals the phylodynamics, plasticity, and paths of tumor evolution. Cell, 2022.
% \bibitem{dynamo}
% Mapping transcriptomic vector fields of single cells. Cell, 2022.
% \bibitem{colosseum}
% Low-cost, scalable, and automated fluid sampling for fluidics applications. HardwareX, 2021.
\bibitem{kb}
Modular and efficient pre-processing of single-cell RNA-seq. Nature Biotechnology, 2021.
\end{thebibliography}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     COLUMN TWO
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\end{minipage}
\hfill
\begin{minipage}[t]{0.69\textwidth}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     RESEARCH
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Experience}
\runsubsection{Software Engineer}
\descript{| Ginkgo Bioworks}
\location{Aug 2022 – Present | Current title: Senior Software Engineer | Boston, MA}
% Individual contributor on the software team identifying gaps in core software, proposing and designing technical solutions, driving cross-team initiatives in an Agile environment, and unblocking high-impact projects.
% Technical lead in one customer project-specific team and mentored junior engineers.
Designed and implemented scalable software solutions for biosecurity and synthetic biology applications in fast-paced Agile environments.
Led cross-functional initiatives to identify core software gaps and unblock high-impact projects.
Technical lead for a customer-specific project team, mentoring junior engineers.
\sectionsep

\runsubsection{Research Assistant}
\descript{| MIT}
\location{Dec 2020 – Jun 2022 | Cambridge, MA}
Studied cancer evolution via synthetic lineage reconstruction using a CRISPR-enabled lineage tracer at the Weissman lab. Developed tools to preprocess metabolic labeling scRNA-seq data \cite{dynast} and high-resolution spatial transcriptomics \cite{spateo}.
\sectionsep

\runsubsection{Software Soln. Engineer Intern}
\descript{| Ginkgo Bioworks}
\location{Jun 2020 – Aug 2020 | Boston, MA}
Migrated in-house mass spectrometry data pipeline to AWS and Apache Airflow.
% Took responsibility of the full development cycle of the project---from conceptualization, making architectural and design decisions to implementation, validation, and release.
Designed and implemented the full development cycle for scalable and reliable pipelines, contributing to a long-term infrastructure upgrade.
\sectionsep

% \runsubsection{Undergraduate Researcher}
% \descript{| Pachter Lab}
% \location{Oct 2019 – Jun 2020 | Pasadena, CA}
% Developed software tools to analyze scRNA-seq data. Most notably \href{https://github.com/pachterlab/kb_python}{kb-python} \cite{kb}.
% \sectionsep

\runsubsection{Software Engineering Intern}
\descript{| Ginkgo Bioworks}
\location{Jun 2019 – Sept 2019 | Boston, MA}
Enhanced data analysis pipelines and backend services to improve stability and performance.
Collaborated with Agile teams to deliver tested, scalable software solutions.
% Improved company's in-house data analysis pipeline and various backend services. Worked collaboratively with other software engineers in an Agile environment to deliver stable, scalable and tested products.
\sectionsep

% \runsubsection{Sternberg Lab}
% \descript{| Undergraduate Researcher}
% \location{Jan 2017 – Jun 2019 | Pasadena, CA}
% Worked with \textbf{\href{https://dangeles.github.io/}{David Angeles PhD.}} and \textbf{\href{http://wormlab.caltech.edu/LabMembers/Paul}{Prof Paul W. Sternberg}} on bioinformatics and molecular biology projects involving the model organism \textit{C. elegans}, most notably Alaska (above).
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     PROJECTS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Projects}

% \runsubsection{Labyrinth}
% \descript{| Software Engineer, Ginkgo Bioworks}
% \location{Jan 2024 - Present | Boston, MA}
% \vspace{\topsep} % Hacky fix for awkward extra vertical space
% \begin{tightemize}
% \item {Internal scientist-centric Python package for easy discovery and retrieval of data specific to a customer project.}
% \item {Developed by independently identifying gaps in core software, proposing and designing a solution, and driving implementation.}
% \end{tightemize}
% \sectionsep

\runsubsection{On-demand Analysis Pipeline}
\descript{| SWE, Ginkgo Bioworks}
\location{Jan 2023 - Dec 2023 | Boston, MA}
\vspace{\topsep} % Hacky fix for awkward extra vertical space
\begin{tightemize}
\item {Refactored an in-house automated analysis pipeline for improved scalability and reliability using Apache Airflow.}
\item {Tech lead for the implementation of a new analysis registry and mentored an intern through its successful delivery.}
\end{tightemize}
\sectionsep

\runsubsection{Biosecurity NGS pipeline}
\descript{| SWE, Ginkgo Bioworks}
\location{Sept 2022 - Jan 2023 | Boston, MA}
\begin{tightemize}
\item {Led productionization of an in-house biosecurity NGS pipeline, significantly enhancing throughput and runtime.}
\item {Orchestration using Apache Airflow and job execution on AWS.}
\end{tightemize}
\sectionsep

% \runsubsection{\href{https://github.com/aristoteleo/spateo-release}{Spateo \faGithub}}
% \descript{| Research Assistant, MIT \cite{spateo}}
% \location{Dec 2021 - Jun 2022 | Cambridge, MA}
% \begin{tightemize}
% \item {Python package for analysis of high-resolution spatial transcriptomics.}
% \item {Developed novel RNA-based cell segmentation approaches.}
% \end{tightemize}
% \sectionsep

% \runsubsection{\href{https://github.com/aristoteleo/dynast-release}{Dynast \faGithub}}
% \descript{| Research Assistant, MIT \cite{dynast}}
% \location{Dec 2020 - Jun 2022 | Cambridge, MA}
% \begin{tightemize}
% \item {An inclusive and efficient preprocessing pipeline for quantifying metabolic labeling-enabled scRNA-seq.}
% \item {Simultaneous quantification of metabolically labeled and spliced transcripts.}
% \end{tightemize}
% \sectionsep

\runsubsection{Mass Spectrometry Pipeline}
\descript{| Software Soln. Engineer Intern, Ginkgo Bioworks}
\location{Jun 2020 – Aug 2020 | Boston, MA}
\begin{tightemize}
\item {Migrated the in-house mass spectrometry pipeline to Linux and AWS, leveraging Apache Airflow for workflow orchestration.}
\item {Retired an internal fork of ProteoWizard by contributing back to upstream.}
\end{tightemize}
\sectionsep

% \runsubsection{Colosseum}
% \descript{| Undergraduate Researcher, Pachter Lab }
% \location{Sept 2019 - Jun 2020 | Pasadena, CA}
% \begin{tightemize}
% \item {A low-cost, modular, open-source automated fluid sampling device for scalable microfluidic applications.}
% \item {Developed its user interface using PyQt5 and installable from Pypi; later migrated to a fully-featured web application written with React.js.}
% \end{tightemize}
% \sectionsep

% \runsubsection{\href{https://github.com/pachterlab/kb_python}{kb-python \faGithub}}
% \descript{| Undergraduate Researcher, Pachter Lab \cite{kb}}
% \location{Oct 2019 – Jun 2020 | Pasadena, CA}
% \begin{tightemize}
% \item A Python wrapper around popular pipeline for scRNA-seq pre-processing.
% \item Pre-processes scRNA-seq data into gene count and RNA velocity matrices.
% \end{tightemize}
% \sectionsep

\runsubsection{NGS Pipeline}
\descript{| Software Engineer Intern, Ginkgo Bioworks }
\location{Jun 2019 - Sept 2019 | Boston, MA}
\begin{tightemize}
\item Migrated in-house NGS pipeline to AWS and Apache Airflow.
\item Implemented pipeline metrics collection with Elasticsearch and Grafana.
\end{tightemize}
\sectionsep

% \runsubsection{Alaska}
% \descript{| Full-Stack Developer, Caltech }
% \location{Jan 2017 - Aug 2019 | Pasadena, CA}
% % \vspace{\topsep} % Hacky fix for awkward extra vertical space
% \begin{tightemize}
% \item Developed a framework for automated, complete RNA-seq analysis.
% \item Presented a poster at the 21st International \textit{C. elegans} Conference.
% \item Available at \textcolor{blue}{ \href{http://alaska.caltech.edu/}{http://alaska.caltech.edu/}} and source code on Github.
% \end{tightemize}
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     EXTRACURRICULARS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Extracurriculars}
% \runsubsection{Yearbook}
% \descript{| Business Manager, Designer, Editor-in-Chief}
% \location{Sept 2016 – Present | Pasadena, CA}
% Established a new advertisement system, shared cloud storage system, job organization and assignment system, and payment reimbursement system. Designed pages and took/edited photos with Adobe software.
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     AWARDS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Awards}
% \begin{tabular}{rl}
% 2018     & Perpall Speaking Competition Semi-finalist \\
% 2018     & Samuel P. and Frances Krown SURF Fellowship  \\
% 2017     & SURF Fellowship \\
% \end{tabular}
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     PUBLICATIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Publications}
% \renewcommand\refname{\vskip -1.5em} % Couldn't get this working from the .cls file
% \begin{thebibliography}{9}
% % \bibitem{kite}
% % Feature demultiplexing for single-cell genomics assays. Co-first author, in preparation.
% \bibitem{colosseum}
% Low-cost, scalable, and automated fluid sampling for fluidics applications. Author, under review.
% \bibitem{kb}
% Modular and efficient pre-processing of single-cell RNA-seq. Author, published.
% \end{thebibliography}

\end{minipage}
\end{document}  \documentclass[]{article}
