%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Deedy - One Page Two Column Resume
% LaTeX Template
% Version 1.2 (16/9/2014)
%
% Original author:
% Debarghya Das (http://debarghyadas.com)
%
% Original repository:
% https://github.com/deedydas/Deedy-Resume
%
% IMPORTANT: THIS TEMPLATE NEEDS TO BE COMPILED WITH XeLaTeX
%
% This template uses several fonts not included with Windows/Linux by
% default. If you get compilation errors saying a font is missing, find the line
% on which the font is used and either change it to a font included with your
% operating system or comment the line out to use the default font.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% TODO:
% 1. Integrate biber/bibtex for article citation under publications.
% 2. Figure out a smoother way for the document to flow onto the next page.
% 3. Add styling information for a "Projects/Hacks" section.
% 4. Add location/address information
% 5. Merge OpenFont and MacFonts as a single sty with options.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% CHANGELOG:
% v1.1:
% 1. Fixed several compilation bugs with \renewcommand
% 2. Got Open-source fonts (Windows/Linux support)
% 3. Added Last Updated
% 4. Move Title styling into .sty
% 5. Commented .sty file.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% Known Issues:
% 1. Overflows onto second page if any column's contents are more than the
% vertical limit
% 2. Hacky space on the first bullet point on the second column.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\documentclass[]{resume}
\usepackage{fontawesome}
\usepackage{hyperref}
\usepackage{fancyhdr}

\pagestyle{fancy}
\fancyhf{}

\begin{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     LAST UPDATED DATE
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\lastupdated
\vspace*{0.5em}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     TITLE NAME
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\namesection{Joseph}{Min}{
Pasadena, CA | \faPhone (626) 240-9851 | \href{mailto:phoenixter96@gmail.com}{\faEnvelope phoenixter96@gmail.com} | \href{https://github.com/lioscro}{\faGithub lioscro} | \href{https://www.linkedin.com/in/lioscro/}{\faLinkedin lioscro}

}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     COLUMN ONE
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{minipage}[t]{0.28\textwidth}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     EDUCATION
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Education}
\subsection{MS, Computer Science}
% \descript{MIT}
\location{MIT | Cambridge, MA}
\sectionsep

\subsection{BS, Biology and \\ Computer Science}
% \descript{Caltech}
\location{Caltech | Pasadena, CA}
% \location{Cum. GPA: 4.0 / 4.3}
%Major GPA: 3.9 / 4.0}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     LINKS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Links}
% Facebook:// \href{https://facebook/dd}{\bf dd} \\
% Github:// \href{https://github.com/deedydas}{\bf deedydas} \\
% LinkedIn://  \href{https://www.linkedin.com/in/debarghyadas}{\bf debarghyadas} \\
% YouTube://  \href{https://www.youtube.com/user/DeedyDash007}{\bf DeedyDash007} \\
% Twitter://  \href{https://twitter.com/debarghya_das}{\bf @debarghya\_das} \\
% Quora://  \href{https://www.quora.com/Debarghya-Das}{\bf Debarghya-Das}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     COURSEWORK
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Coursework}
% \subsection{Undergraduate}
% \location{Biology}
% Regulation of Gene Expression \\
% Cell Biology \\
% Genetics \\
% Bioinformatics {\footnotesize \textit{(Planned)}}\\
% \sectionsep
% \location{Computer Science}
% Programming Methods \\
% Decidability and Tractability \\
% Communication Networks \\
% Algorithms \\
% Learning systems {\footnotesize \textit{(machine learning)}}\\
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     SKILLS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Skills}

\subsection{Programming}
\begin{itemize}[leftmargin=*]
    \itemsep0pt \parskip0pt \parsep0pt
    \item{\textbf{Python}}
    \item{\textbf{Javascript}, Typescript}
    \item{Ruby on Rails}
    \item{SQL}
    \item{C\# (.NET Framework)}
    \item{C, C++}
    % \item{R}
\end{itemize}

\subsection{Software}
\begin{itemize}[leftmargin=*]
    \itemsep0pt \parskip0pt \parsep0pt
    \item{\textbf{Git, CI/CD}}
    \item{\textbf{Docker}}
    \item{\textbf{Apache Airflow}}
    % \item{\textbf{Scanpy, AnnData}}
    % \item{\textbf{samtools, bcftools}}
    \item{\textbf{Django}}
    \item{\textbf{AWS (Batch, EC2, S3, Lambda)}}
    % \item{\textbf{Nextflow}}
    \item{\textbf{React.js}}
    % \item{Redis}
    \item{Snowflake}
    \item{Kubernetes}
    % \item{PostgreSQL}
    % \item{SQLAlchemy}
    \item{Kafka}
\end{itemize}

\subsection{Other}
\begin{itemize}[leftmargin=*]
    \itemsep0pt \parskip0pt \parsep0pt
    % \item{Software design \& architecture}
    \item{Software development}
    \item{Orchestration \& distributed computing}
    % \item{Bioinformatics/Computational biology (NGS)}
    \item{Project management}
    \item{Technical communication}
\end{itemize}

% \subsection{Molecular Biology}
% \textbullet{PCR}\\
% \textbullet{Cloning}\\
% \textbullet{Purification}\\
% \sectionsep



% \location{Over 5000 lines:}
% Java \textbullet{}   Shell \textbullet{} Python \textbullet{} Javascript \\
% OCaml \textbullet{} Matlab \textbullet{} Rails \textbullet{} \LaTeX\ \\
% \location{Over 1000 lines:}
% C \textbullet{} C++ \textbullet{} CSS \textbullet{} PHP \textbullet{} Assembly \\
% \location{Familiar:}
% AS3 \textbullet{} iOS \textbullet{} Android \textbullet{} MySQL
% \sectionsep

\section{Publications}
\renewcommand\refname{\vskip -1.5em} % Couldn't get this working from the .cls file
\begin{thebibliography}{9}
\bibitem{spateo}
Spateo: multidimensional spatiotemporal modeling of single-cell spatial transcriptomics. in press, 2024
\bibitem{dynast}
Dynast: Inclusive and efficient quantification of metabolically labeled transcripts in single cells. MIT, 2022
% \bibitem{kp}
% Lineage tracing reveals the phylodynamics, plasticity, and paths of tumor evolution. Cell, 2022.
% \bibitem{dynamo}
% Mapping transcriptomic vector fields of single cells. Cell, 2022.
% \bibitem{colosseum}
% Low-cost, scalable, and automated fluid sampling for fluidics applications. HardwareX, 2021.
\bibitem{kb}
Modular and efficient pre-processing of single-cell RNA-seq. Nature Biotechnology, 2021.
\end{thebibliography}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%     COLUMN TWO
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\end{minipage}
\hfill
\begin{minipage}[t]{0.69\textwidth}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     RESEARCH
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Experience}
\runsubsection{Senior Software Engineer}\\
% \descript{| Ginkgo Bioworks}
\location{Aug 2022 – Present | Ginkgo Bioworks, Boston, MA}
\vspace{\topsep} % Hacky fix for awkward extra vertical space
\begin{tightemize}
    \item{Developed backend software solutions for synthetic biology applications in a highly collaborative and Agile environment.}
    \item{Served as technical lead for a software engineering team embedded in a major pharmaceutical project and owned multiple bioinformatic pipelines.}
    % \item{Owned the development and maintenance of multiple bioinformatic pipelines, including those for variant calling and copy number estimation.}
    \item{Acted as a core engineer and SME for a bioinformatic analysis execution pipeline processing thousands of jobs daily on cloud infrastructure (AWS).}
    \item{Led cross-functional efforts involving multiple engineers to enhance the reliability, scalability, and throughput of computational biology workload orchestration.}
    \item{Mentored junior engineers and interns in software engineering best practices.}
\end{tightemize}
\sectionsep

\runsubsection{Software Developer}\\
% \descript{| Weissman Lab, MIT}
\location{Aug 2020 – Jun 2022 | MIT, Cambridge, MA}
\begin{tightemize}
    \item{Developed software tools for cancer evolution and synthetic biology research.}
    \item{Processed terabyte-scale multi-modal (single-cell) sequencing datasets using LSF in a distributed, parallelized environment.}
    \item{Designed and implemented computational algorithms for lineage tracing, preprocessing and spatial transcriptomics.}
\end{tightemize}
% Studied cancer evolution via synthetic lineage reconstruction using a CRISPR-enabled lineage tracer at the Weissman lab. Developed tools to preprocess metabolic labeling scRNA-seq data \cite{dynast} and high-resolution spatial transcriptomics \cite{spateo}.
\sectionsep

% \runsubsection{Software Soln. Engineer Intern}
% \descript{| Ginkgo Bioworks}
% \location{Jun 2020 – Aug 2020 | Boston, MA}
% Migrated in-house mass spectrometry data pipeline to AWS and Apache Airflow.
% % Took responsibility of the full development cycle of the project---from conceptualization, making architectural and design decisions to implementation, validation, and release.
% Designed and implemented the full development cycle for scalable and reliable pipelines, contributing to a long-term infrastructure upgrade.
% \sectionsep

% \runsubsection{Undergraduate Researcher}
% \descript{| Pachter Lab}
% \location{Oct 2019 – Jun 2020 | Pasadena, CA}
% Developed software tools to analyze scRNA-seq data. Most notably \href{https://github.com/pachterlab/kb_python}{kb-python} \cite{kb}.
% \sectionsep

% \runsubsection{Software Engineering Intern}
% \descript{| Ginkgo Bioworks}
% \location{Jun 2019 – Sept 2019 | Boston, MA}
% Enhanced data analysis pipelines and backend services to improve stability and performance.
% Collaborated with Agile teams to deliver tested, scalable software solutions.
% % Improved company's in-house data analysis pipeline and various backend services. Worked collaboratively with other software engineers in an Agile environment to deliver stable, scalable and tested products.
% \sectionsep

% \runsubsection{Sternberg Lab}
% \descript{| Undergraduate Researcher}
% \location{Jan 2017 – Jun 2019 | Pasadena, CA}
% Worked with \textbf{\href{https://dangeles.github.io/}{David Angeles PhD.}} and \textbf{\href{http://wormlab.caltech.edu/LabMembers/Paul}{Prof Paul W. Sternberg}} on bioinformatics and molecular biology projects involving the model organism \textit{C. elegans}, most notably Alaska (above).
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     PROJECTS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Projects}

\runsubsection{Data Abstraction Layer}\\
% \descript{| Ginkgo Bioworks}
\location{Apr 2024 - Apr 2025 | Ginkgo Bioworks}
% \vspace{\topsep} % Hacky fix for awkward extra vertical space
\begin{tightemize}
    \item{Developed a Python-based data abstraction layer to streamline access and analysis across the company’s data stack, hiding the complexity of underlying data sources.}
    \item{Enabled scientists to query and analyze data through a unified API, removing the need to understand underlying data models and reducing time-to-insight by 30\%.}
    \item{Integrated diverse data sources, including SQL databases (via REST and GraphQL APIs), Snowflake, and S3 data lakes from distributed systems.}
\end{tightemize}
\sectionsep


% \runsubsection{Genomic Edit Tracking}\\
% % \descript{| Ginkgo Bioworks}
% \location{Apr 2024 - Mar 2025 | Ginkgo Bioworks}
% % \vspace{\topsep} % Hacky fix for awkward extra vertical space
% \begin{tightemize}
%     \item{Built a system with Git-like version control to track and verify millions of genomic edits across 100k samples, enabling branching, merging, and rebasing of genome edits.}
%     \item{Owned the bioinformatic pipeline to verify genomic edits by graph alignment, including optimizations that reduced runtime by 50\% and costs by 90\%.}
%     \item{Developed a high-throughput Nextflow pipeline to detect unintended edits (variants) and predict their impact, processing over 5k samples per hour.}
%     \item{Designed and implemented an ETL process with Apache Airflow to load data into a Snowflake warehouse for user access and downstream analysis.}
% \end{tightemize}
% \sectionsep

\runsubsection{Distributed Analysis Orchestration Pipeline}\\
% \descript{| Ginkgo Bioworks}
\location{Sept 2022 - Apr 2024 | Ginkgo Bioworks}
% \vspace{\topsep} % Hacky fix for awkward extra vertical space
\begin{tightemize}
    \item{Led the development of the first company-wide analysis registry, improving analysis discoverability and tracking. Reduced onboarding time for new analyses by 50\% and mentored an intern through its successful delivery.}
    \item{Optimized resource allocation and job scheduling in cloud infrastructure (AWS Batch) by implementing auto-scaling, cutting costs by 30\% with no performance impact.}
    \item{Enhanced distributed computing in cloud infrastructure for an in-house biosecurity NGS pipeline, increasing throughput by 1000\% and reducing costs by 50\%.}
\end{tightemize}
\sectionsep

\runsubsection{Scientific Research Software}\\
\location{Aug 2020 - Jun 2022 | MIT}
\begin{tightemize}
    \item{Developed \href{https://github.com/aristoteleo/spateo-release}{Spateo \faGithub}, a Python package for high-resolution spatial transcriptomics analysis, pioneering mRNA-based cell segmentation.}
    \item{Developed \href{https://github.com/aristoteleo/dynast-release}{Dynast \faGithub}, an advanced preprocessing pipeline for biological sequencing, delivering more comprehensive results at a fraction of the computational cost of existing methods.}
    \item{Enhanced \href{https://github.com/YosefLab/Cassiopeia}{Cassiopeia \faGithub}, a single-cell lineage reconstruction tool, reducing runtime by 30\% and implementing novel graph reconstruction algorithms.}
\end{tightemize}


% \runsubsection{Biosecurity NGS pipeline}\\
% % \descript{| SWE, Ginkgo Bioworks}
% \location{Sept 2022 - Jan 2023 | Ginkgo Bioworks}
% \begin{tightemize}
%     \item{Led productionization of an in-house biosecurity NGS pipeline, by optimizing its use of distributed computing infrastructure using Apache Airflow and AWS Batch.}
%     \item{Increased pipeline throughput by 1000\% and reduced costs by 50\%.}
% \end{tightemize}
% \sectionsep

% \runsubsection{\href{https://github.com/aristoteleo/spateo-release}{Spateo \faGithub}}
% \descript{| Research Assistant, MIT \cite{spateo}}
% \location{Dec 2021 - Jun 2022 | Cambridge, MA}
% \begin{tightemize}
% \item {Python package for analysis of high-resolution spatial transcriptomics.}
% \item {Developed novel RNA-based cell segmentation approaches.}
% \end{tightemize}
% \sectionsep

% \runsubsection{\href{https://github.com/aristoteleo/dynast-release}{Dynast \faGithub}}
% \descript{| Research Assistant, MIT \cite{dynast}}
% \location{Dec 2020 - Jun 2022 | Cambridge, MA}
% \begin{tightemize}
% \item {An inclusive and efficient preprocessing pipeline for quantifying metabolic labeling-enabled scRNA-seq.}
% \item {Simultaneous quantification of metabolically labeled and spliced transcripts.}
% \end{tightemize}
% \sectionsep

% \runsubsection{Mass Spectrometry Pipeline}
% \descript{| Software Soln. Engineer Intern, Ginkgo Bioworks}
% \location{Jun 2020 – Aug 2020 | Boston, MA}
% \begin{tightemize}
% \item {Migrated the in-house mass spectrometry pipeline to Linux and AWS, leveraging Apache Airflow for workflow orchestration.}
% \item {Retired an internal fork of ProteoWizard by contributing back to upstream.}
% \end{tightemize}
% \sectionsep

% \runsubsection{Colosseum}
% \descript{| Undergraduate Researcher, Pachter Lab }
% \location{Sept 2019 - Jun 2020 | Pasadena, CA}
% \begin{tightemize}
% \item {A low-cost, modular, open-source automated fluid sampling device for scalable microfluidic applications.}
% \item {Developed its user interface using PyQt5 and installable from Pypi; later migrated to a fully-featured web application written with React.js.}
% \end{tightemize}
% \sectionsep

% \runsubsection{\href{https://github.com/pachterlab/kb_python}{kb-python \faGithub}}
% \descript{| Undergraduate Researcher, Pachter Lab \cite{kb}}
% \location{Oct 2019 – Jun 2020 | Pasadena, CA}
% \begin{tightemize}
% \item A Python wrapper around popular pipeline for scRNA-seq pre-processing.
% \item Pre-processes scRNA-seq data into gene count and RNA velocity matrices.
% \end{tightemize}
% \sectionsep

% \runsubsection{NGS Pipeline}
% \descript{| Software Engineer Intern, Ginkgo Bioworks }
% \location{Jun 2019 - Sept 2019 | Boston, MA}
% \begin{tightemize}
% \item Migrated in-house NGS pipeline to AWS and Apache Airflow.
% \item Implemented pipeline metrics collection with Elasticsearch and Grafana.
% \end{tightemize}
% \sectionsep

% \runsubsection{Alaska}
% \descript{| Full-Stack Developer, Caltech }
% \location{Jan 2017 - Aug 2019 | Pasadena, CA}
% % \vspace{\topsep} % Hacky fix for awkward extra vertical space
% \begin{tightemize}
% \item Developed a framework for automated, complete RNA-seq analysis.
% \item Presented a poster at the 21st International \textit{C. elegans} Conference.
% \item Available at \textcolor{blue}{ \href{http://alaska.caltech.edu/}{http://alaska.caltech.edu/}} and source code on Github.
% \end{tightemize}
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     EXTRACURRICULARS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Extracurriculars}
% \runsubsection{Yearbook}
% \descript{| Business Manager, Designer, Editor-in-Chief}
% \location{Sept 2016 – Present | Pasadena, CA}
% Established a new advertisement system, shared cloud storage system, job organization and assignment system, and payment reimbursement system. Designed pages and took/edited photos with Adobe software.
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     AWARDS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Awards}
% \begin{tabular}{rl}
% 2018     & Perpall Speaking Competition Semi-finalist \\
% 2018     & Samuel P. and Frances Krown SURF Fellowship  \\
% 2017     & SURF Fellowship \\
% \end{tabular}
% \sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     PUBLICATIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \section{Publications}
% \renewcommand\refname{\vskip -1.5em} % Couldn't get this working from the .cls file
% \begin{thebibliography}{9}
% % \bibitem{kite}
% % Feature demultiplexing for single-cell genomics assays. Co-first author, in preparation.
% \bibitem{colosseum}
% Low-cost, scalable, and automated fluid sampling for fluidics applications. Author, under review.
% \bibitem{kb}
% Modular and efficient pre-processing of single-cell RNA-seq. Author, published.
% \end{thebibliography}

\end{minipage}
\end{document}  \documentclass[]{article}
